This patch is needed until MX1 architecture is integrated in 
plat-mcx (2.6.31 ?).

Index: linux-2.6.29/arch/arm/mach-imx/include/mach/usb.h
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ linux-2.6.29/arch/arm/mach-imx/include/mach/usb.h	2009-05-01 13:51:56.000000000 +0200
@@ -0,0 +1,25 @@
+/*
+ *	Copyright (C) 2008 Darius Augulis <augulis.darius@gmail.com>
+ *
+ *	This program is free software; you can redistribute it and/or modify
+ *	it under the terms of the GNU General Public License as published by
+ *	the Free Software Foundation; either version 2 of the License, or
+ *	(at your option) any later version.
+ *
+ *	This program is distributed in the hope that it will be useful,
+ *	but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *	GNU General Public License for more details.
+ */
+
+#ifndef __ASM_ARCH_MXC_USB
+#define __ASM_ARCH_MXC_USB
+
+struct imxusb_platform_data {
+	int (*init)(struct device *);
+	int (*exit)(struct device *);
+};
+
+void set_imx_usb_info(struct imxusb_platform_data *hard_imx_usb_info);
+
+#endif /* __ASM_ARCH_MXC_USB */
